.github/workflows/bench-testnet.yaml

```diff
@@ -0,0 +1,32 @@
+name: "Bench Devnet"
+
+on:
+  # Remove this before merging, we only want devnet tests on master
```

Comment:
> [!NOTE]
> Remove this before merging so that we only run the devnet workflow on master builds

Reply:
@maschad I'd say it's best to run only for the [release PRs](https://github.com/FuelLabs/fuels-ts/pull/3174).

We'll still have the chance to review it before merging and identify which PR caused a regression — which, by the way, I think we should disable for now and switch to _observation_ mode until we are comfortable with our numbers and thresholds; then, we can re-enable it. 

With this, we should not have any blocked PRs and releases that can't be merged because of regressions, like what is happening in:
 - https://github.com/FuelLabs/fuels-ts/pull/3174

<br/>

![regression](https://github.com/user-attachments/assets/93c46fa1-8775-4469-88ff-34a650d62cdd)

Reply:
@arboleya I agree with you re: release PRs 

I assume you mean by _observation mode_ is that the workflow still runs but we can still go ahead and merge to prevent PRs from being blocked. As we have been doing already

<img width="908" alt="Screenshot 2024-09-17 at 8 56 13 AM" src="https://github.com/user-attachments/assets/87d6f9b6-1c31-4bac-9b58-8d984b4c9ce0">